docs/en/inference/convert_tutorial.md

```diff
@@ -166,13 +181,35 @@ paddle2onnx \
 The `input_shape_dict` in the parameter can generally be viewed by opening the inference model using the [Netron](https://github.com/lutzroeder/netron),
 or found in the code in [tools/export_model. py](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/tools/export_model.py) above.
 
-#### 2.3 ONNX -> OM
+#### 2.3 ONNX -> MindIR
+
+The `converter_lite` can be used to convert the ONNX into a MindSpore Lite MindIR. For detailed usage tutorials,
+please refer to [Offline Conversion of Inference Models](https://www.mindspore.cn/lite/docs/en/r2.0/use/cloud_infer/converter_tool.html)。
+
+The conversion command is as follows:
+
+```shell
+converter_lite \
+    --saveType=MINDIR \
+    --NoFusion=false \
+    --fmk=ONNX \
+    --device=Ascend \
+    --modelFile=det_db.onnx \
+    --outputFile=det_db_output \
+    --configFile=config.txt
+```
+
+The conversion process is completely the same as the [MindOCR models](#1-mindocr-models),
+except that `--fmk` needs to specify that the input is the ONNX, which will not be repeated here.
```

Comment:
`--fmk` --> `--fmt` ??

except that `--fmt`  needs to be set as `ONNX`.

Reply:
the original format of the input model is fmk in lite converter_tool. https://www.mindspore.cn/lite/docs/en/r2.0/use/converter_tool.html